lib/registry.js

```diff
@@ -118,7 +119,17 @@ util.inherits(RegistryItem, Object);
 Object.freeze(RegistryItem);
 Object.freeze(RegistryItem.prototype);
 
-
+/**
+ * Get the path to system's reg.exe. Useful when another reg.exe is added to the PATH
+ * Implemented only for Windows
+ */
+function getRegExePath() {
+    if (process.platform === 'win32') {
+        return path.join(process.env.windir, 'system32', 'reg.exe');
```

Comment:
In my really brief testing it looks like /reg:32/64 is backwards compatible in Win7 32-bit (just always returns the 32-bit hive).